system/jlib/jexcept.cpp

```diff
@@ -1452,6 +1452,44 @@ void printStackReport(__int64 startIP)
 
 //---------------------------------------------------------------------------------------------------------------------
 
+bool getAllStacks(StringBuffer &output)
+{
+#ifdef __linux__
+    StringBuffer exePath;
+    char *_exePath = exePath.reserveTruncate(NAME_MAX + PATH_MAX +1);
+    ssize_t len = readlink("/proc/self/exe", _exePath, NAME_MAX + PATH_MAX +1);
+    if (-1 == len)
+    {
+        output.append("Unable to capture stacks");
+        return false;
+    }
+    _exePath[len] = '\0';
```

Comment:
Should this use queryCurrentProcessPath()?

Reply:
yep, didn't know of that func. but changed to use in latest commit.